packages/apps/reputation-oracle/server/src/modules/auth/auth.error.filter.ts

```diff
@@ -9,16 +9,16 @@ import { Request, Response } from 'express';
 
 import {
   AuthError,
-  DuplicatedUserError,
+  DuplicatedUserEmailError,
   InvalidOperatorSignupDataError,
-} from './auth.errors';
+} from './auth.error';
 
 type AuthControllerError =
   | AuthError
-  | DuplicatedUserError
+  | DuplicatedUserEmailError
   | InvalidOperatorSignupDataError;
 
-@Catch(AuthError, DuplicatedUserError, InvalidOperatorSignupDataError)
+@Catch(AuthError, DuplicatedUserEmailError, InvalidOperatorSignupDataError)
```

Comment:
Auth service now also throws `DuplicatedUserAddressError`, so we should catch it here as well